File: ago-dic-2018/José Horacio Góngora Treviño/Practica_4/Libro.java

public class Libro 
{
	private String nombre;
	private int pag;
	
	public Libro(String nombre, int pag)
	{
		super();
		this.nombre=nombre;
		this.pag=pag;		
	}
	
	public String getNombre()
	{
		return nombre;
	}
	
	public void setNombre(String nombre)
	{
		this.nombre=nombre;
	}
	
	public int getPag()
	{
		return pag;
	}
	
	public void setPag(int pag)
	{
		this.pag=pag;
	}
	
	public String toString()
	{
		return nombre + "," + pag;
	}
}

File: ago-dic-2020/Luis Enrique Cázares Martínez/Parcial 1.5/libro.py
class libro():
    def __init__(self, nombre, paginas):
        self.nombre = nombre
        self.paginas = paginas

    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre

    def getPaginas(self):
        return self.paginas

    def setPaginas(self, paginas):
        self.paginas = paginas;

    def __str__(self):
        return f"Libro -> {self.nombre} Paginas -> {self.paginas}"